docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -145,31 +311,96 @@ Without medical complications
 Baseline Coverage Data
 ++++++++++++++++++++++++
 
+Let us assume program coverage is a stand-in value of 40%. Cmam and Csam = 0.4; 1-Cmam and 1-Csam = 0.6 (This will be updated). 
+
 .. todo::
 
   We have been in contact with EMOH but seems like they only have DHIS data which does not give us coverage rate. We will dig a bit more into the literature, or reach out to CIFF/UNICEF since UNICEF is the implementation partner for treatment in Ethiopia. 
 
 .. list-table:: Baseline coverage data
-  :widths: 15 15 15 15 15
+  :widths: 10 15 20 10 15
   :header-rows: 1
 
   * - Location
     - Subpopulation
     - Coverage parameter
     - Value
     - Note
-  * - 
-    - 
-    - 
-    - 
-    - 
+  * - Ethiopia
+    - Kids 0-6 months with MAM 
+    - Cmam_0_6
+    - 0.4 ~ currently stand-in until we find better data
+    - Coverage of TSFP for breastfeeding mothers among 0-6 month kids with MAM
+  * - Ethiopia
+    - Kids 0-6 months with any SAM 
+    - Csam_0-6
+    - 0.4 ~ currently stand-in until we find better data
+    - Coverage of SC for kids 0-6 months with any SAM
+  * - Ethiopia
+    - Kids 6-59 months with MAM 
+    - Cmam_6_59
+    - 0.4 ~ currently stand-in until we find better data
+    - Coverage of TSFP for kids 6-59 months with MAM
+  * - Ethiopia
+    - Kids 6-59 months with SAM with oedema
+    - Csam_6_59_sc 
+    - 0.4 ~ Currently stand-in until we find better data
+    - Coverage of SC for SAM kids with medical complications (oedema)
+  * - Ethiopia
+    - Kids 6-59 months with SAM without complications (no oedema)
+    - Csam_6_59_otp 
+    - 0.4 ~ currently stand-in until we find better data
+    - Coverage of SC for kids with un-complicated SAM
+
+.. _3.0:
 
 Vivarium Modeling Strategy
 --------------------------
 
+.. image:: treatment_diagram.svg
+   :alt: Compartmental diagram with treatment
+
+
+For **MAM** 0-6 months
+
+ - :math:`r3 =  r3_{ux}\times(1-C_{mam_0-6}) + r3_{tx}\times(C_{mam_0-6})`
```

Comment:
so we will need to look for effect sizes for the different treatments, and then they all effect either **i** or **r**?

Reply:
YES ! I will need to look at treatment outcomes from programmes for those 5 treatment interventions.